bool are_motors_moving
bool are_wheels_moving
bool is_animating
bool is_being_held
bool is_button_pressed
bool is_carrying_block
bool is_charging
bool is_cliff_detected
bool is_docking_to_marker
bool is_falling
bool is_head_in_pos
bool is_in_calm_power_mode # when vector is sleeping or charging
bool is_lift_in_pos
bool is_on_charger
bool is_pathing
bool is_picked_up
bool is_robot_moving
